### PR TITLE
gpasswd: add page

### PR DESCRIPTION
**Pull request by msdlisper**
gpasswd: add page

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/gpasswd.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# gpasswd
+
+> Administer "/etc/group" and "/etc/gshadow". Administer linux group.
```
Comment (agnivade) [inline pages/linux/gpasswd.md]: Is the second line necessary ? The command only manages those 2 files. I think "Administer "/etc/group" and "/etc/gshadhow" files" sounds clear enough.

File: pages/linux/gpasswd.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Define group administrator(s):
+
+`sudo gpasswd -A {{user1[,user2]}} {{group}}`
```
Comment (agnivade) [inline pages/linux/gpasswd.md]: Please remove the []. And just user "user1,user2"

File: pages/linux/gpasswd.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# gpasswd
+
+> Administer "/etc/group" and "/etc/gshadow". Administer linux group.
+
+- Define group administrator(s):
```
Comment (agnivade) [inline pages/linux/gpasswd.md]: Remove ()

File: pages/linux/gpasswd.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`sudo gpasswd -A {{user1[,user2]}} {{group}}`
+
+- Set the list of group member:
```
Comment (agnivade) [inline pages/linux/gpasswd.md]: members

File: pages/linux/gpasswd.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Create a password for the named group:
+
+`gpasswd {{group}}`
+
+- Add a user to the named group:
+
+`gpasswd -a {{user}} {{group}}`
+
+- Remove a user to the named group:
```
Comment (agnivade) [inline pages/linux/gpasswd.md]: _from_ the named group

File: pages/linux/gpasswd.md (hunk 1 above)
Comment (msdlisper) [reply pages/linux/gpasswd.md]: I agree.

**Pull request merged**